src/data/team.ts

```diff
@@ -73,6 +74,12 @@ export const getMembers = memoize(async () => {
           member.manager = omit(manager, "manager");
         }
       }
+      const reports = parsed
+        .filter((m) => m.reports_to === member.name)
+        .map((report) => omit(report, ["manager", "reports"]));
+      if (reports.length > 0) {
+        member.reports = reports;
+      }
```

Comment:
This is the logic update that populates the reports field on the member object. Essentially we're iterating through the parsed members from the gsheet, filtering an array of those members who have a `reports_to` field that matches the current member name, and omitting recursive fields like `manager` and `reports` (because the final data structure needs to be JSON serializable meaning no loops).